<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartnerCardWithButton</class>
 <widget class="QWidget" name="PartnerCardWithButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: #fff;
	color: #000;
	font-size: 15px; 
	padding: 0px
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QFrame" name="frame">
       <property name="styleSheet">
        <string notr="true">.QFrame{
	border: 2px solid black
}</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="PartnerTypeAndName">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>PartnerTypeAndName</string>
            </property>
            <property name="autoRepeatInterval">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BossName">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>BossName</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PhoneNumber">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>PhoneNumber</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Rank">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Rank</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item alignment="Qt::AlignmentFlag::Qt::AlignmentFlag::AlignTop">
         <widget class="QPushButton" name="DiscountPercentage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>DiscountPercentage</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="GetOrdersListButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #67BA80;   border-radius: 10px</string>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
